DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System;
+
+    public static class DisplayStringHelper
+    {
+        private const string Divider = " / ";
+        private static readonly string[] Units = { "B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB" };
+
+        public static string GenerateNumberWithLimitDisplayString(int number, int limit)
+        {
+            return limit == -1 ? number.ToString() : number + Divider + limit;
+        }
+
+        public static string GenerateBytesLimitDisplayString(long number, long limit)
+        {
+            return GenerateBytesDisplayString(number) + Divider + GenerateBytesDisplayString(limit);
+        }
+
+        private static string GenerateBytesDisplayString(long byteCount)
+        {
+            if (byteCount == 0)
+            {
+                return 0 + Units[0];
+            }
+
+            var bytes = Math.Abs(byteCount);
+            var place = Convert.ToInt32(Math.Floor(Math.Log(bytes, 1024)));

```

Comment:
This is a lot of nesting.. Do you think it's worth splitting this up a bit (might make scanning over the algorithm a bit nicer too than in the old system)